components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIGatewayManager.java

```diff
@@ -145,6 +149,10 @@ public synchronized static APIGatewayManager getInstance() {
         if (api.getGatewayLabels() != null) {
             for (Label label : api.getGatewayLabels()) {
                 Environment environment = getEnvironmentFromLabel(label);
+                if (debugEnabled) {
+                    log.debug("API with " + api.getId() + " and label " + label + " is publishing to the "
```

Comment:
Change the debug message to API with api ID is publishing to gateway with label

Reply:
https://github.com/wso2/carbon-apimgt/pull/9019/commits/6b56e3e82804d8002c3efac4f6c5e1de4286823d